applications/nrf_desktop/configuration/common/led_state.h

```diff
@@ -69,12 +69,7 @@ enum led_id_nrf_desktop {
 
 #define LED_UNAVAILABLE 0xFF
 
-#if (defined(CONFIG_BT_PERIPHERAL) && defined(CONFIG_BT_CENTRAL)) || \
-    (!defined(CONFIG_BT_PERIPHERAL) && !defined(CONFIG_BT_CENTRAL))
-#error Device should be either Bluetooth peripheral or central
-#endif
```

Comment:
Why have you removed this check? Do we have it in another place now (Kconfig description)?

Reply:
Currently these Zephyr's Kconfigs were replaced by nRF Desktop's application-specific Kconfigs (`CONFIG_DESKTOP_BT_PERPIHERAL`, `CONFIG_DESKTOP_BT_CENTRAL`) - the new approach is more accurate (as nRF Desktop peripheral you could support central role, but you would not work as nRF Desktop central, because the application-specific roles are mutually exclusive). You event cannot set both Kconfigs simultaneously because of dependencies.